README.md

```diff
@@ -29,6 +29,9 @@
     <img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square"
          alt="Standard Codestyle">
   </a>
+  <a href="https://discord.gg/m62rFAV4NU">
+  <img src=" https://img.shields.io/discord/782374564054564875" alt="Discord badge" />
```

Comment:
There appears to be a space in the `src`, I think causing an issue with the image loading.